openfisca_web_api_preview/loader/__init__.py

```diff
@@ -13,11 +13,17 @@ def extract_description(items):
         }
 
 
-def build_data(country_package_name):
+def build_data(country_package_name, extensions):
     tax_benefit_system = build_tax_benefit_system(country_package_name)
     country_package_metadata = tax_benefit_system.get_package_metadata()
+
+    if extensions is not None:
+        for extension in extensions:
+            tax_benefit_system.load_extension(extension)
+
     parameters = build_parameters(tax_benefit_system)
     variables = build_variables(tax_benefit_system, country_package_metadata)
+
```

Comment:
Why adding a line break here ?

Reply:
To separate the work on the package carcass from the inner building. 😅 
Removed.